cli/src/config.rs

```diff
@@ -56,17 +56,23 @@ pub struct Config {
     ignore_certs_cli: bool,
     #[serde(deserialize_with = "default_option_bool")]
     ignore_certs: bool,
+    #[serde(rename = "organization")]
+    org: Option<String>,
+    #[serde(skip)]
+    org_cli: Option<String>,
```

Comment:
nit: Seems a bit strange that this struct mixes values that are persisted between runs and values that are provided as arguments to the CLI, but not that big of a deal.

Reply:
This is already true for the certs stuff. I'd argue it's not that unusual to keep layered non-persistent configuration options stored together with the persistent information. This way users cannot accidentally ignore the config override which makes dealing with this sort of layered configuration significantly easier.